city_scrapers/spiders/il_pollution_control.py

```diff
@@ -0,0 +1,164 @@
+                title=title,
+                description="",  # Too inconsistent to parse accurately
+                classification=self._parse_classification(title),
+                start=self._parse_start(item),
+                end=None,
+                all_day=item["IsFullDay"],
+                time_notes="",
+                location=self._parse_location(item),
+                links=list(),
+                source=self._parse_source(item),
+            )
+
+            meeting["links"] = self._parse_links(meeting)
+            meeting["status"] = self._parse_status(meeting, item)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_status(self, meeting, item):
+        text = " ".join([item['CalendarTypeDesc'], item['Description']]).lower()
+        if "cancel" in text:
+            return CANCELLED
+        elif meeting["start"] < datetime.now():
+            return PASSED
+        else:
+            return TENTATIVE
+
+    def _parse_classification(self, title):
+        """Parse or generate classification from allowed options."""
+        if "Board" in title:
+            return BOARD
+        elif "Seminar" in title:
+            return FORUM
+        else:
+            return NOT_CLASSIFIED
+
+    def _parse_start(self, item):
+        return datetime.strptime(item["StartDateTime"], '%m/%d/%Y %I:%M:%S %p')
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        text = " ".join([item['Description'], item['Location']]).lower()
+        if "thompson" in text:
+            return {
+                "address":
+                    "James R. Thompson Center - 100 W. Randolph St. Suite 11-500, Chicago, IL 60601",  # noqa
+                "name": "Chicago IPCB Office",
+            }
+        elif "springfield" in text or "llinois pollution control board" in text:
+            return {
+                "address": "1021 N. Grand Ave. E. - Room 1244 N, Springfield, IL 62702",
+                "name": "Springfield IPCB Office",
+            }
+        elif "sangamo room" in text:
+            return {
+                "address": "1021 N. Grand Ave. E. - Sangamo Room, Springfield, IL 62702",
+                "name": "Illinois EPA",
+            }
+        else:
+            return {
+                "address": "",
+                "name": "",
+            }
+
+    def _parse_links(self, meeting):
+        """ Associate Meeting objects with previously-scraped links """
+        key = meeting['start'].date()
+        if key in self.link_map:
+            return [{"href": self.link_map[key], "title": "Meeting Minutes - {}".format(key)}]
+
+    def _parse_source(self, item):
+        """Parse or generate source."""
+        rel_url = scrapy.Selector(text=item["Description"]).xpath(".//a/@href").get()
+        if rel_url:
+            return urljoin("https://{}".format(self.allowed_domains[0]), rel_url)
+        else:
+            return self.json_url
```

Comment:
We should default this to the HTML page calendar since that would be easier for people to refer back to than the raw JSON https://pcb.illinois.gov/ClerksOffice/Calendar

Reply:
Yep that makes much more sense.